spyder_kernels/console/kernel.py

```diff
@@ -972,16 +946,54 @@ def _set_config_option(self, option, value):
         except Exception:
             pass
 
-    def _set_mpl_inline_rc_config(self, option, value):
+    def _set_inline_config_option(self, option, value):
         """
-        Update any of the Matplolib rcParams given an option and value.
+        Update InlineBackend given an option and value.
+
+        As parameters:
+            option: config option; one of 'close_figures', 'figure_formats',
+                'print_figure_kwargs', or 'rc'.
+            value: value of the option
```

Comment:
Please rewrite this part in the Numpy docstring format, i.e.

```python
"""
Parameters
----------
option: str
    ...
"""
```

Reply:
```suggestion
        Parameters
        ----------
        option: str
            Configuration option. One of 'close_figures', 'figure_formats',
            'print_figure_kwargs', or 'rc'.
        value: str | dict
            Value of the option. 
```